docs/content/developing.md

```diff
@@ -43,3 +43,29 @@ instead, or the build command can call that process.
 
 `spin build --up` can be used to start the application after the build process
 finishes for all application components.
+
+## Component `workdir`
+
+By default, the `command` to build a component is executed in the manifest's
+directory. This can be changed. For example, assume a component is located in
+subdirectory `deep`:
+
+```bash
+.
+├── deep
+│   ├── Cargo.toml
+│   └── src
+│       └── lib.rs
+└── spin.toml
+```
+
+To run the build `command` in directory `deep`, set the component's `workdir`:
+
+```toml
+[component.build]
+command = "cargo build --target wasm32-wasi --release"
+workdir = "deep"
+```
+
+Note that `workdir` must be a relative path, specifying an absolute path leads
```

Comment:
Would it make sense to mention here that `workdir` operates relative to the `spin.toml`? Or is that obvious given the Note.

Reply:
Great point, I think it's worth mentioning it here.

Reply:
True, it is mentioned now.